backend/tracim_backend/lib/utils/authorization.py

```diff
@@ -361,7 +361,7 @@ def check(self, tracim_context: TracimContext) -> bool:
 can_create_content = ContentTypeCreationChecker(content_type_list)
 # comments
 is_comment_owner = CommentOwnerChecker()
-can_delete_comment = OrAuthorizationChecker(
+can_edit_comment = OrAuthorizationChecker(
     AndAuthorizationChecker(is_contributor, is_comment_owner), is_workspace_manager
```

Comment:
pr is approved but this `is_contributor` is confusing me

Reply:
Shouldn't it be `is_contributor`? The checker requires the logged user to:
- be the original comment owner/author AND be a contributor (so have write access) in the comment's space
- OR be a manager of the workspace.

Reply:
the access right are correct, i re-discuss with @gbonaspetti about it: https://github.com/tracim/tracim/issues/4367#issuecomment-810175311
Only user A/admin of the workspace should be able to edit user A comment unless user A is reader, reader are not able at all to do edition in workspace.
The is_contributor check, allow for contributor+ level user to acess.

Reply:
good, I resolve the conversation